### PR TITLE
Docs: Added App links to README.md

### LINKED ISSUE #550: Add link of the app in the README.md
[F-Droid ](https://f-droid.org/en/packages/com.eventyay.attendee/)
[Play Store](https://play.google.com/store/apps/details?id=com.eventyay.attendee)

### PR DESCRIPTION
**Pull request by DhruvThakker**
Fixes #550

### CODE REVIEW

**Comment by simarsingh24** [comment]
please check the fdroid link and confirm if its working

File: README.md [hunk 1]
```diff
@@ -6,6 +6,14 @@
 
 An events app to discover events happening around the world using the Open Event Platform
 
+Application is available at:
+
+Google Play Store [here](https://play.google.com/store/apps/details?id=com.eventyay.attendee) and
+F-Droid [Here](https://f-droid.org/en/packages/com.eventyay.attendee/).
```
Comment (nikit19) [inline README.md]: Looks good. Just change the above 2 lines to `  [Google Play Store](https://play.google.com/store/apps/details?id=com.eventyay.attendee) and
 [F-Droid](https://f-droid.org/en/packages/com.eventyay.attendee/)`

**Comment by DhruvThakker** [comment]
> please check the fdroid link and confirm if its working

@simarsingh24 F-Droid link working. Checked.

**Comment by DhruvThakker** [comment]
@nikit19 Changes have been implemented. Please review.

**Pull request merged**